client/src/components/SignupForm.js

```diff
@@ -94,99 +95,20 @@ const SignupForm = () => {
           />
         </Form.Field>
       </Form.Group>
-      <Button
-        inverted
-        color="red"
+      <PinkButton
         type="submit"
-        style={{ fontSize: "18px", marginTop: "1.5em" }}
         disabled={
           !(
-            userFormData.username &&
             userFormData.email &&
+            userFormData.password &&
```

Comment:
Did you mean for there to be two "userFormData.password"? -> "userFormData.password && userFormData.password"

```suggestion
            userFormData.password &&
```

Reply:
I'll fix it

Reply:
Fixed